src/queries/text.ts

```diff
@@ -19,7 +20,10 @@ import type {
   QueryByQuery,
 } from './makeQueries';
 import type { CommonQueryOptions } from './options';
-import { getHostComponentNames } from '../helpers/host-component-names';
+
+beforeEach(() => {
+  resetToDefaults();
+});
```

Comment:
I don't understand these lines, this file is not a test file. Maybe an issue occurred during the rebase?